realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -135,6 +136,41 @@ class RealmTransformer extends Transform {
 
         def toc = System.currentTimeMillis()
         logger.info "Realm Transform time: ${toc-tic} milliseconds"
+
+        sendAnalytics(inputs, inputModelClasses)
+    }
+
+    /**
+     * Sends the analytics
+     * @param inputs the inputs provided by the Transform API
+     * @param inputModelClasses a list of ctClasses describing the Realm models
+     */
+    private static sendAnalytics(Collection<TransformInput> inputs, List<CtClass> inputModelClasses) {
+        def containsKotlin = false
+        inputs.each {
```

Comment:
Do we really need to loop through files to do this? Can't we just check to see if the kotlin plugin is applied just like when we select APT/KAPT ?